<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(255, 0, 0);
	border-top-left-radius: 15px; 
border-top-right-radius: 15px; 
border-bottom-left-radius: 15px; 
border-bottom-right-radius: 15px; 



}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>43</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>43</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
background-color:rgba(245, 245, 245, 95%);
border-top-left-radius:10;
border-top-right-radius:10;


}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
border:none;
background-color:rgba(245, 245, 245, 0%);


}</string>
           </property>
           <widget class="QPushButton" name="pushButton_11">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{


border-radius:10px;
border-image: url(:/fork1.png) 0 0 0 0;
}
QPushButton:hover{
border-image: url(:/fork2.png) 0 0 0 0;

}
QPushButton:pressed{
border-image: url(:/fork3.png) 0 0 0 0;

}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_12">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>0</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{


border-radius:10px;
border-image: url(:/small1.png) 0 0 0 0;
}
QPushButton:hover{
border-image: url(:/small2.png) 0 0 0 0;

}
QPushButton:pressed{
border-image: url(:/small3.png) 0 0 0 0;

}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_13">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>0</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{


border-radius:10px;
border-image: url(:/big1.png) 0 0 0 0;
}
QPushButton:hover{
border-image: url(:/big2.png) 0 0 0 0;

}
QPushButton:pressed{
border-image: url(:/big3.png) 0 0 0 0;

}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>57</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{

background-color:rgba(245, 245, 245, 0%);


}</string>
           </property>
           <property name="text">
            <string>CutDown</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{
background-color:rgba(245, 245, 245, 0%);



}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
background-color:rgba(245, 245, 245, 95%);



}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{

background-color:rgba(245, 245, 245, 0%);



}</string>
           </property>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>300</width>
              <height>70</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget{

background-color:rgba(0,0,0, 0%);
}</string>
            </property>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>50</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
background-color:rgba(245, 245, 245, 0%);



}</string>
             </property>
             <property name="text">
              <string>   分享</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_7">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>0</y>
               <width>51</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:2px solid red; 
border-radius:20px;
border-image: url(:/ya3.png) 0 0 0 0;
}
QPushButton:pressed{
border-image: url(:/ya2.png) 0 0 0 0;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>93</x>
               <y>50</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
background-color:rgba(245, 245, 245, 0%);



}</string>
             </property>
             <property name="text">
              <string>  压缩</string>
             </property>
            </widget>
            <widget class="QFrame" name="frame">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>281</y>
               <width>120</width>
               <height>80</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QPushButton" name="pushButton_10">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>51</width>
                <height>51</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
border:2px solid red; 
border-radius:20px;

border-image: url(:/jie2.png) 0 0 0 0;
}
QPushButton:pressed{
border-image: url(:/jie3.png) 0 0 0 0;

}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>32</x>
               <y>50</y>
               <width>31</width>
               <height>21</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
background-color:rgba(245, 245, 245, 0%);



}</string>
             </property>
             <property name="text">
              <string>打开</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_5">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>0</y>
               <width>51</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:2px solid red; 
border-radius:20px;

border-image: url(:/fx2.png) 0 0 0 0;
}
QPushButton:pressed{
border-image: url(:/fx3.png) 0 0 0 0;

}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>165</x>
               <y>50</y>
               <width>41</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
background-color:rgba(245, 245, 245, 0%);



}</string>
             </property>
             <property name="text">
              <string>  解压</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_6">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>0</y>
               <width>51</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:2px solid red; 
border-radius:20px;

border-image: url(:/jie2.png) 0 0 0 0;
}
QPushButton:pressed{
border-image: url(:/jie3.png) 0 0 0 0;

}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>0</y>
               <width>51</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:2px solid red; 
border-radius:20px;
border-image: url(:/jia1.png) 0 0 0 0;
}
QPushButton:pressed{
border-image: url(:/jia3.png) 0 0 0 0;

}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget{

background-color:rgba(245, 245, 245, 0%);



}</string>
           </property>
           <widget class="QPushButton" name="pushButton_15">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>51</width>
              <height>51</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border:2px solid red; 
border-radius:20px;

border-image: url(:/1.png) 0 0 0 0;
}
QPushButton:hover{
border-image: url(:/1_1.png) 0 0 0 0;

}

QPushButton:pressed{
border-image: url(:/1_2.png) 0 0 0 0;

}


</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>1</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>1</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <property name="minimumSize">
              <size>
               <width>260</width>
               <height>260</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget{
border-image:url(:/jiahao.png) 0 0 0 0;


}</string>
             </property>
             <widget class="QPushButton" name="pushButton">
              <property name="geometry">
               <rect>
                <x>1</x>
                <y>1</y>
                <width>260</width>
                <height>261</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="label_9">
              <property name="geometry">
               <rect>
                <x>1</x>
                <y>1</y>
                <width>261</width>
                <height>261</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">QLabel{
font: 20pt &quot;.SF NS Text&quot;;

}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:35pt; color:#8a8a8a;&quot;&gt;拖拽文件到这里&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QWidget" name="widget_11" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>1</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QWidget" name="widget_12" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>1</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_10" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
border:none;
background-color:white;


}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
border:none;
background-color:rgba(253,253,253,99%);
border-bottom-left-radius:10; 
border-bottom-right-radius:10; 


}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>30</height>
            </size>
           </property>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>241</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>100M将要被解压</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>241</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>0M将要被压缩</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
